src/encoded/__init__.py

```diff
@@ -128,8 +128,9 @@ def main(global_config, **local_config):
     # settings['snovault.jsonld.terms_namespace'] = 'https://www.encodeproject.org/terms/'
     settings['snovault.jsonld.terms_prefix'] = 'encode'
     # set auth0 keys
-    settings['auth0.secret'] = os.environ.get("Auth0Secret")
-    settings['auth0.client'] = os.environ.get("Auth0Client")
+    if not settings.get('auth0.client', None):
+        settings['auth0.secret'] = os.environ.get("Auth0Secret")
+        settings['auth0.client'] = os.environ.get("Auth0Client")
```

Comment:
In the code you offer, if someone provides the secret variable but not the client variable but not the other, neither will be seen. I think the behavior will be less mysterious if you do:
```suggestion
    settings['auth0.client'] = settings.get('auth0.client', os.environ.get('Auth0Client'))
    settings['auth0.secret'] = settings.get('auth0.secret', os.environ.get('Auth0Secret'))
```